projects/opendr_ws/src/opendr_perception/README.md

```diff
@@ -95,237 +190,581 @@ Reference images should be placed in a defined structure like:
+   - `--buffer_size BUFFER_SIZE`: length of audio and video in seconds, (default=`3.6`)
+   - `--model_path MODEL_PATH`: if given, the pretrained model will be loaded from the specified local path, otherwise it will be downloaded from an OpenDR FTP server
+
+4. Default output topics:
+   - Detection messages: `/opendr/audiovisual_emotion`
+   
+   For viewing the output, refer to the [notes above.](#notes)
+
+----
+## Audio input
+
+### Speech Command Recognition ROS Node
 
 A ROS node for recognizing speech commands from an audio stream using MatchboxNet, EdgeSpeechNets or Quadratic SelfONN models, pretrained on the Google Speech Commands dataset.
-Assuming that the OpenDR catkin workspace has been sourced, the node can be started with:
-```shell
-rosrun perception speech_command_recognition.py INPUT_AUDIO_TOPIC
-```
-The following optional arguments are available:
-- `--buffer_size BUFFER_SIZE`: set the size of the audio buffer (expected command duration) in seconds, default value **1.5**
-- `--model MODEL`: choose the model to use: `matchboxnet` (default value), `edgespeechnets` or `quad_selfonn`
-- `--model_path MODEL_PATH`: if given, the pretrained model will be loaded from the specified local path, otherwise it will be downloaded from an OpenDR FTP server
 
-The predictions (class id and confidence) are published to the topic `/opendr/speech_recognition`.
-**Note:** EdgeSpeechNets currently does not have a pretrained model available for download, only local files may be used.
+You can find the speech command recognition ROS node python script [here](./scripts/speech_command_recognition_node.py) to inspect the code and modify it as you wish to fit your needs. 
+The node makes use of the toolkit's speech command recognition tools: 
+[EdgeSpeechNets tool](../../../../src/opendr/perception/speech_recognition/edgespeechnets/edgespeechnets_learner.py), [MatchboxNet tool](../../../../src/opendr/perception/speech_recognition/matchboxnet/matchboxnet_learner.py), [Quadratic SelfONN tool](../../../../src/opendr/perception/speech_recognition/quadraticselfonn/quadraticselfonn_learner.py) 
+whose documentation can be found here: 
+[EdgeSpeechNet docs](../../../../docs/reference/edgespeechnets.md), [MatchboxNet docs](../../../../docs/reference/matchboxnet.md), [Quadratic SelfONN docs](../../../../docs/reference/quadratic-selfonn.md).
 
-## Voxel Object Detection 3D ROS Node
+#### Instructions for basic usage:
 
-A ROS node for performing Object Detection 3D using PointPillars or TANet methods with either pretrained models on KITTI dataset, or custom trained models.
-The predicted detection annotations are pushed to `output_detection3d_topic` (default `output_detection3d_topic="/opendr/detection3d"`).
+1. Start the node responsible for publishing audio. Remember to modify the input topics using the arguments in step 2, if needed.
```

Comment:
When mentioning step's number, sometimes you put the dot `2.`, sometimes not. Personally I think the number is sufficient but we should be consistent, both options are fine either way

Reply:
Thanks i missed this when checking for consistency across the document.